### PR TITLE
adding ontomap file and fixing CITEseq on Marcel's suggestions

### PR DESCRIPTION
**Pull request by drighelli**
fixing suggestions from the previous PR https://github.com/waldronlab/SingleCellMultiModal/pull/55

### CODE REVIEW

File: R/CITEseq.R [hunk 1]
```diff
@@ -192,32 +160,30 @@
...
 #'
 #' mae <- CITEseq(DataType="cord_blood", dry.run=FALSE)
 #' experiments(mae)
-#'
 CITEseq <- function(DataType=c("cord_blood", "peripheral_blood"), modes="*",
-    version="1.0.0", dry.run=TRUE, verbose=TRUE,
-    DataClass=c("MultiAssayExperiment", "SingleCellExperiment"),
-    ...)
+                version="1.0.0", dry.run=TRUE, verbose=TRUE, 
+                DataClass=c("MultiAssaiExperiment", "SingleCellExperiment"),
```
Comment (LiNk-NY) [inline R/CITEseq.R:178]: This should be `MultiAssayExperiment`

File: R/CITEseq.R [hunk 2]
```diff
@@ -256,76 +222,58 @@ CITEseq <- function(DataType=c("cord_blood", "peripheral_blood"), modes="*",
...
         scrna <- scrna[,(colnames(scrna) %in% commonsamp)]
         scadt <- scadt[,(colnames(scadt) %in% commonsamp)]
-
-        sce <- SingleCellExperiment::SingleCellExperiment(
-            list(counts=scrna), altExps=list(scADT=scadt, scHTO=schto)
-        )
-    } else if (length(mae)==2) {
+        
+        sce <- SingleCellExperiment::SingleCellExperiment(list(counts=scrna),
+                                                          altExps=list(scADT=scadt,
```
Comment (LiNk-NY) [inline R/CITEseq.R:239]: There is code here that is just "noise" for the PR. If there are no significant changes, please do not change the spacing / formatting. It makes it harder to review.

File: R/CITEseq.R [hunk 3]
```diff
@@ -226,8 +192,8 @@ CITEseq <- function(DataType=c("cord_blood", "peripheral_blood"), modes="*",
             ## Add here other CITE-seq datasets based on DataType identifier
             { stop("Unrecognized CITE-seq dataset name: ", DataType) }
         )
-
-        if (dataClass=="SingleCellExperiment") return(.CITEseqMaeToSce(mae))
+        
+        if(dataClass=="SingleCellExperiment") return(.CITEseqMaeToSce(mae))
```
Comment (LiNk-NY) [inline R/CITEseq.R:196]: There is code here that is just "noise" for the PR. If there are no significant changes, please do not change the spacing / formatting.

**Review by LiNk-NY — APPROVED** [review]
The formatting should not change if there are no code improvements. There are a few lines that are adding blanks or moving code to a new line. These are distracting and make it more difficult to review. More importantly, there are ways to reduce cyclomatic complexity by using a single operation (e.g., `Map`) regardless of the length input list (e.g., either one or two) but I will leave those for future improvement. Thanks!

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
Hi Dario, @drighelli 
Can you address the first comment in the previous review?
I think that one is the only necessary change.
Thank you.

**Comment by drighelli** [comment]
Hi Marcel, @LiNk-NY 

I thought the comment was already fixed, am I missing something?

**Comment by LiNk-NY** [comment]
Hi Dario, @drighelli  
I took care of it.

**Comment by drighelli** [comment]
oh ok! sorry about that!

**Pull request merged**